### PR TITLE
Use https://oss.sonatype.org/content/groups/public/ instead of http:/…

### PR DESCRIPTION
**Pull request by dbaxa**
…/oss.sonatype.org/content/groups/public/ .

Signed-off-by: David Black <dblack@atlassian.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 		<repository>
 			<id>Sonatype-public</id>
 			<name>SnakeYAML repository</name>
-			<url>http://oss.sonatype.org/content/groups/public/</url>
+			<url>https://oss.sonatype.org/content/groups/public/</url>
```
Comment (dbaxa) [inline pom.xml]: I am not actually use that this repository is used.

Comment (yannmh) [inline pom.xml]: You are right, this can be completely removed. Would you like to add a commit for that? 🙇

Comment (dbaxa) [inline pom.xml]: Done.

**Comment by yannmh** [comment]
Thanks @dbaxa 🙇

**Pull request merged**